lib/Fmt_ast.ml

```diff
@@ -2417,6 +2432,8 @@ and fmt_expression c ?(box = true) ?(pro = noop) ?eol ?parens
       let override = is_override flag in
       let outer_parens = has_attr && parens in
       let inner_parens = has_attr || parens in
+      (* [let open] extensions are external for now. *)
+      (* let ext = attributes.attrs_extension in *)
```

Comment:
I don't understand this comment. The two syntaxes for let open extensions seem to work as before.

Reply:
I see.